woudc_data_registry/models.py

```diff
@@ -818,7 +841,7 @@ def unpack_station_names(rows):
         if name.startswith('\\x'):
             name = decode_hex(name[2:])[0].decode('utf-8')
             row['name'] = name
-            row['identifier'] = ':'.join([station, name])
+            row['station_name_id'] = ':'.join([station, name])
         if name not in tracker:
             tracker[name] = row
         else:
```

Comment:
Now that we no longer have station name dates, this whole `else` block can be taken out.